conf/livy-client.conf.template

```diff
@@ -55,7 +55,7 @@
 
 # Address for the RSC driver to connect back with it's connection info.
 # livy.rsc.launcher.address =
-# livy.rsc.launcher.port = -1
+# livy.rsc.launcher.port.range = 10000~10110
```

Comment:
Could you add one more for this configuration, especially about whether start/end port is inclusive and exclusive